setup.cfg

```diff
@@ -5,10 +5,6 @@ test=pytest
 addopts = --verbose
 python_files = tests/**
 
-[bdist_wheel]
-# this flags the wheel as being python 2 and 3 compatible
-universal = 1
```

Comment:
This actually labels the binary distribution as python 2 compatible, which it isn't